src/main/java/org/n52/wacodis/javaps/algorithms/LandCoverClassificationAlgorithm.java

```diff
@@ -183,20 +183,18 @@ public String getResultNamePrefix() {
     private AbstractCommandValue preprocessOpticalImages() throws WacodisProcessingException {
 //        InputDataPreprocessor imagePreprocessor = new Sentinel2Preprocessor(false, this.getNamingSuffix());
         InputDataPreprocessor imagePreprocessor = new GptPreprocessor(FilenameUtils.concat(this.config.getGpfDir(), GPF_FILE), TIFF_EXTENSION, this.getNamingSuffix());
-
+        
         try {
             // Download satellite data
-            File sentinelFile = sentinelDownloader.downloadSentinelFile(
-                    this.opticalImagesSource,
-                    this.config.getWorkingDirectory());
-            this.sentinelProduct = ProductIO.readProduct(sentinelFile.getPath());
+            //File sentinelFile = sentinelDownloader.downloadSentinelFile(this.opticalImagesSource,this.config.getWorkingDirectory());
```

Comment:
Please remove comment again in order to activate Sentinel file downloading.

Reply:
Removed the comment